hardware/mcp3008/pimcp3008.js

```diff
@@ -1,12 +1,12 @@
 
 module.exports = function(RED) {
     "use strict";
-    var fs = require('fs');
+    var fs2 = require('fs');
```

Comment:
Not sure why we need this duplication ?

Reply:
This is a holdover from debugging.
The line can remain original